Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.4.4'
+ruby '2.5.0'
```

Comment:
Aha, cool! I didn't know Ruby 2.5 was out! 

In any case if you add changes you should be able to push up additional commits to the same PR. If not, please close this one and open a new one. Thank you so much!!!

Reply:
Hi @jywarren I installed ruby 2.4.4 on my laptop and set the local ruby version for this app to the same. I've re-pushed my branch with the ruby version in the Gemfile back at 2.4.4. But the change is not update is not updating in my PR 🤷‍♀️

Reply:
Hmm, I'm not seeing an extra commit listed here -- are you sure you pushed to `msarit:gMapsKeyToEnvParam` ?

Great emoji work! 😄 

Just curious how you came across our project? We're working on improving out outreach so we love to hear how new people find us. Thanks!

Reply:
I sure did! It's strange ... maybe I'll just edit the file in here.

I heard about you all from Dev.to 😄